### PR TITLE
Release new preview

### PR DESCRIPTION
**Pull request by jamarw**
1. Merging into master from develop.

### CODE REVIEW

File: docs-preview/package.json [hunk 1]
```diff
@@ -66,89 +62,10 @@
...
-      "binaries": [
-        "./MarkdocsService"
-      ],
-      "installTestPath": "./.markdocs/MarkdocsService"
-    }
-  ]
+    "chai": "^4.2.0",
+    "markdown-it": "^8.4.2",
+    "markdown-it-include": "^1.1.0",
+    "tslint": "^5.16.0",
```
Comment (bharney-msft) [inline docs-preview/package.json]: I think tslint should be a dev dependency?

Comment (jamarw) [reply docs-preview/package.json]: Good call

File: docs-preview/package.json [hunk 2]
```diff
@@ -62,93 +58,14 @@
...
-        "x86_64"
-      ],
-      "binaries": [
-        "./MarkdocsService"
-      ],
-      "installTestPath": "./.markdocs/MarkdocsService"
-    }
-  ]
-}
+    "chai": "^4.2.0",
```
Comment (nschonni) [inline docs-preview/package.json]: This is probably also a devDependency

Comment (jamarw) [reply docs-preview/package.json]: Good call

**Pull request merged**